scripts/smartereye.sh

```diff
@@ -0,0 +1,71 @@
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5Gui.so.5
+sudo ln -s libQt5Gui.so.5.12.2 libQt5Gui.so.5
+
+sudo cp /apollo/third_party/camera_library/smartereye/qtlib5.12.2/libQt5Network.so.5.12.2 /usr/lib/x86_64-linux-gnu/
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5Network.so.5
+sudo ln -s libQt5Network.so.5.12.2 libQt5Network.so.5
+
+sudo cp /apollo/third_party/camera_library/smartereye/qtlib5.12.2/libQt5OpenGL.so.5.12.2 /usr/lib/x86_64-linux-gnu/
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5OpenGL.so.5
+sudo ln -s libQt5OpenGL.so.5.12.2 libQt5OpenGL.so.5
+
+sudo cp /apollo/third_party/camera_library/smartereye/qtlib5.12.2/libQt5Positioning.so.5.12.2 /usr/lib/x86_64-linux-gnu/
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5Positioning.so.5
+sudo ln -s libQt5Positioning.so.5.12.2 libQt5Positioning.so.5
+
+sudo cp /apollo/third_party/camera_library/smartereye/qtlib5.12.2/libQt5PrintSupport.so.5.12.2 /usr/lib/x86_64-linux-gnu/
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5PrintSupport.so.5
+sudo ln -s libQt5PrintSupport.so.5.12.2 libQt5PrintSupport.so.5
+
+sudo cp /apollo/third_party/camera_library/smartereye/qtlib5.12.2/libQt5Qml.so.5.12.2 /usr/lib/x86_64-linux-gnu/
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5Qml.so.5
+sudo ln -s libQt5Qml.so.5.12.2 libQt5Qml.so.5
+
+sudo cp /apollo/third_party/camera_library/smartereye/qtlib5.12.2/libQt5Quick.so.5.12.2 /usr/lib/x86_64-linux-gnu/
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5Quick.so.5
+sudo ln -s libQt5Quick.so.5.12.2 libQt5Quick.so.5
+
+sudo cp /apollo/third_party/camera_library/smartereye/qtlib5.12.2/libQt5Sensors.so.5.12.2 /usr/lib/x86_64-linux-gnu/
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5Sensors.so.5
+sudo ln -s libQt5Sensors.so.5.12.2 libQt5Sensors.so.5
+
+sudo cp /apollo/third_party/camera_library/smartereye/qtlib5.12.2/libQt5Svg.so.5.12.2 /usr/lib/x86_64-linux-gnu/
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5Svg.so.5
+sudo ln -s libQt5Svg.so.5.12.2 libQt5Svg.so.5
+
+sudo cp /apollo/third_party/camera_library/smartereye/qtlib5.12.2/libQt5WebChannel.so.5.12.2 /usr/lib/x86_64-linux-gnu/
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5WebChannel.so.5
+sudo ln -s libQt5WebChannel.so.5.12.2 libQt5WebChannel.so.5
+
+sudo cp /apollo/third_party/camera_library/smartereye/qtlib5.12.2/libQt5Widgets.so.5.12.2 /usr/lib/x86_64-linux-gnu/
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5Widgets.so.5
+sudo ln -s libQt5Widgets.so.5.12.2 libQt5Widgets.so.5
+
+sudo cp /apollo/third_party/camera_library/smartereye/qtlib5.12.2/libQt5X11Extras.so.5.12.2 /usr/lib/x86_64-linux-gnu/
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5X11Extras.so.5
+sudo ln -s libQt5X11Extras.so.5.12.2 libQt5X11Extras.so.5
+
+sudo cp /apollo/third_party/camera_library/smartereye/qtlib5.12.2/libQt5XcbQpa.so.5.12.2 /usr/lib/x86_64-linux-gnu/
+sudo rm -f /usr/lib/x86_64-linux-gnu/libQt5XcbQpa.so.5
+sudo ln -s libQt5XcbQpa.so.5.12.2 libQt5XcbQpa.so.5
+
```

Comment:
Please check with @ycool and @Capri2014 to see if this is an appropriate way to manage libraries.

Reply:
No, we talked offline and I do not think this is a proper way of doing it, this might potentially affect other applications depending on qt also.